### PR TITLE
fixed CQ for bitmessageqt.dialogs module

### PR DESCRIPTION
**Pull request by coolguy-cell**
pylint fixes for bitmessageqt.dialogs module

### CODE REVIEW

**Comment by PeterSurda** [comment]
Maybe change module docstring?

File: src/bitmessageqt/dialogs.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 src/bitmessageqt/dialogs.py
 ===========================
 """
-
+# pylint: disable=relative-import,too-few-public-methods
```
Comment (PeterSurda) [inline src/bitmessageqt/dialogs.py:5]: `relative-import` isn't needed. Please make sure your code quality tools use the configuration from the project root.

File: src/bitmessageqt/dialogs.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 src/bitmessageqt/dialogs.py
```
Comment (PeterSurda) [inline src/bitmessageqt/dialogs.py:2]: `"""Custom dialog classes"""`

**Comment by coolguy-cell** [comment]
Done.

**Comment by coolguy-cell** [comment]
Done

**Pull request merged**